### ISSUE TITLE
remind user about field type/format consistentcy

### ISSUE THREAD

**Issue by xinase** [microsoft/OCR-Form-Tools#515]
**Is your feature request related to a problem? Please describe.**
I have seen one user mistakenly mark a "currency" field as "alphanumeric",  as a result, the ".", and "," characters are removed.  

expect: when user set a type/format of a field value, we can do a sanity check of current data, and give warning if there is a mismatch.

**Comment by stew-ro** [comment]
I think there should also be a confirmation displayed to the user after they change a field type or format. The confirmation should let the user know the change applies to all documents. There is a similar confirmation for the user when they try to rename or delete a tag:
![image](https://user-images.githubusercontent.com/60453211/91417750-19e35c80-e806-11ea-8c8e-25e65804bf35.png)
![image](https://user-images.githubusercontent.com/60453211/91417793-27004b80-e806-11ea-9b87-46f7bf415738.png)

**Comment by xinase** [comment]
yes,  I'd like to see the confirmation text to be more specific,  e.g. if the value is "10/23/2010" and user want to change it to a "currency",  we should see a clear warning with the actual value.

**Comment by stew-ro** [comment]
To clarify, I think it should show a confirmation even when type or format **is** consistent. But, the more detailed message explained above when the for when the type or format **isn't** consistent should be displayed
